### PR TITLE
Feature/hyp 146

### PR DESCRIPTION
**Pull request by n3op2**
# What

Another simple one, adding action handler so it on click returns all row data along with stories and test update

Screenshot 2024-03-05 at 10 58 05 am
Screenshot 2024-03-05 at 10 56 35 am
Screenshot 2024-03-05 at 10 56 25 am

### CODE REVIEW

File: src/Table/Table.stories.tsx [hunk 1]
```diff
@@ -33,12 +34,14 @@ Default.args = {
     ['2011-01-01T00:00:00.000Z', 'example-2', <button>button</button>],
     ['2012-01-01T00:00:00.000Z', 'example-3', <h3>heading #3</h3>],
   ],
+  action: (item) => console.log(item),
```
Comment (mattdean-digicatapult) [inline src/Table/Table.stories.tsx:37]: this shouldn't do a `console.log` surely? This is what `action` is for right?

**Review by mattdean-digicatapult — DISMISSED** [review]

File: src/Table/Table.stories.tsx (hunk 1 above)
Comment (dc-andysign) [inline src/Table/Table.stories.tsx:37]: Do we want console logs in the final thing or are you adding this to prevent the linter from showing red lines ?

**Review by dc-andysign — DISMISSED** [review]
Just one possible request and then I think this can be merged ..

File: src/Table/Table.stories.tsx (hunk 1 above)
Comment (mattdean-digicatapult) [reply src/Table/Table.stories.tsx:37]: it's in the story so it's not like it's exported. My preference would be to use `action` like we do elsewhere unless there's a good reason not to. @n3op2 ?

Comment (n3op2) [reply src/Table/Table.stories.tsx:37]: oops, it was me testing:) yes, should have been an action, did not update the default variant

**Review by dc-andysign — APPROVED** [review]
👍

**Pull request merged**